core/templates/domain/topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service.spec.ts

```diff
@@ -13,98 +13,84 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for CreatorDashboardBackendApiService.
+ * @fileoverview Unit tests for Topics and skills dashboard.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+import { TopicsAndSkillsDashboardBackendApiService } from
+  // eslint-disable-next-line max-len
+  '../topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service';
 
-import { TranslatorProviderForTests } from 'tests/test.extras';
+describe('Topics and Skills Dashboard backend API service', ()=> {
+  // eslint-disable-next-line max-len
+  let TopicsAndSkillsDashboardBackendApiService :TopicsAndSkillsDashboardBackendApiService = null;
```

Comment:
We should only disable eslint when absolutely necessary (like in the import statement above). Otherwise, break around whitespace so it fits:

```typescript
let TopicsAndSkillsDashboardBackendApiService:
  TopicsAndSkillsDashboardBackendApiService = null;
```